src/koaApi.coffee

```diff
@@ -77,10 +79,27 @@ exports.setupApp = (done) ->
   app.use route.get '/visualizer/events/:receivedTime', visualizer.getLatestEvents
   app.use route.get '/visualizer/sync', visualizer.sync
 
-  app.use route.get '/metrics', metrics.getGlobalLoadTimeMetrics
-  app.use route.get '/metrics/status', metrics.getGlobalStatusMetrics
-  app.use route.get '/metrics/:type/:channelId', metrics.getChannelMetrics
-  
+# -- Old metrics Routes Purposely commented out for Backward Compatibility
+  app.use route.get '/mongometrics', metrics.getGlobalLoadTimeMetrics
+  app.use route.get '/mongometrics/status', metrics.getGlobalStatusMetrics
+  app.use route.get '/mongometrics/:type/:channelId', metrics.getChannelMetrics
+# -- End of Old Metrics Routes --
+
+# -- New metrics Routes --
+  app.use route.get '/metrics', if statsd_instance.enabled then statsd.retrieveTransactionCountPerHour else metrics.getGlobalLoadTimeMetrics
+  app.use route.get '/metrics/status', if statsd_instance.enabled then statsd.fetcGlobalStatusMetrics else metrics.getGlobalStatusMetrics
+  app.use route.get '/metrics/:type/:channelId', if statsd_instance.enabled then statsd.retrieveChannelMetrics else metrics.getChannelMetrics
+  app.use route.get '/metrics/load-time', if statsd_instance.enabled then statsd.retrieveAverageLoadTimePerHour else metrics.getGlobalLoadTimeMetrics
+
+# -- StatsD only metrics routes --
+  app.use route.get '/statsd', statsd.retrieveTransactionCountPerHour
+  app.use route.get '/statsd/status', statsd.fetcGlobalStatusMetrics
+  app.use route.get '/statsd/:type/:channelId', statsd.retrieveChannelMetrics
+  app.use route.get '/statsd/load-time', statsd.retrieveAverageLoadTimePerHour
+  app.use route.get '/statsd/transactions/:channelId/:status', statsd.transactionsPerChannelPerHour
```

Comment:
It looks like the last one `transactionsPerChannelPerHour` is the only statd only route. Can we not remove lines 95-98 as they are covered in 89-92.